### ISSUE TITLE
Error: The object is not enumerable when DefaultAccess = ScriptAccess.None

### ISSUE THREAD

**Issue by vbornand** [microsoft/ClearScript#471]
If I try to enumerate an array when `DefaultAccess = ScriptAccess.None`. I get the error:
`Error: The object is not enumerable`

Sample:
```cs
using Microsoft.ClearScript;
using Microsoft.ClearScript.V8;

namespace TestArray;

internal class Program
{
    static void Main(string[] args)
    {
        using (var engine = new V8ScriptEngine())
        {
            engine.DefaultAccess = ScriptAccess.None;
            engine.AddHostObject("house", new House());
            engine.Execute("for (const vi of house.ValuesInt) { house.WriteLine(vi); }");
        }
     }
}

[DefaultScriptUsage(ScriptAccess.Full)]
public class House
{
    public House() {
        ValuesInt = new List<int> { 1, 2 };
    }

    public List<int> ValuesInt { get; set; }

    public void WriteLine(string value)
    {
        Console.WriteLine(value);
    }
}
```

I found a workaround with a `CustomAttributeLoader` which add the ScriptAccess.None attribute on every resources excepted for the resource *ClearScript.Core*.

**Comment by ClearScriptLib** [comment]
Hi @vbornand,

That's a bug; we'll have a fix in the next release.

Thanks for reporting it!

**Comment by ClearScriptLib** [comment]
Fixed in [Version 7.3.7](https://github.com/microsoft/ClearScript/releases/tag/7.3.7).
